boards/shields/st_b_cams_omv_mb1683/doc/index.rst

```diff
@@ -0,0 +1,115 @@
+| 11         | DCMI_D4         | 12         | DCMI_D5      |
++------------+-----------------+------------+--------------+
+| 13         | DCMI_D2         | 14         | DCMI_D3      |
++------------+-----------------+------------+--------------+
+| 15         | DCMI_D0         | 16         | DCMI_D1      |
++------------+-----------------+------------+--------------+
+| 17         | PWR_EN / LED1   | 18         | RESET#       |
++------------+-----------------+------------+--------------+
+
+ZIF connector CN5
+*****************
+
++------------+--------------+------------+--------------+
+| Pin number | Description  | Pin number | Description  |
++============+==============+============+==============+
+| 1          | 3V3          | 16         | DCMI_PIXCK   |
++------------+--------------+------------+--------------+
+| 2          | GND          | 17         | GND          |
++------------+--------------+------------+--------------+
+| 3          | I2C_SCL      | 18         | SPI_MISO     |
++------------+--------------+------------+--------------+
+| 4          | I2C_SDA      | 19         | SPI_CS       |
++------------+--------------+------------+--------------+
+| 5          | RESET#       | 20         | DCMI_D7      |
++------------+--------------+------------+--------------+
+| 6          | PWR_EN / LED1| 21         | DCMI_D6      |
++------------+--------------+------------+--------------+
+| 7          | SHUTTER      | 22         | DCMI_D5      |
++------------+--------------+------------+--------------+
+| 8          | GND          | 23         | DCMI_D4      |
++------------+--------------+------------+--------------+
+| 9          | PULLDOWN / LED2 | 24      | DCMI_D3      |
++------------+--------------+------------+--------------+
+| 10         | Camera_CLK   | 25         | DCMI_D2      |
++------------+--------------+------------+--------------+
+| 11         | 3V3          | 26         | DCMI_D1      |
++------------+--------------+------------+--------------+
+| 12         | DCMI_VSYNC   | 27         | DCMI_D0      |
++------------+--------------+------------+--------------+
+| 13         | 5V (RSU)     | 28         | SPI_MOSI     |
++------------+--------------+------------+--------------+
+| 14         | DCMI_HSYNC   | 29         | SPI_CLK      |
++------------+--------------+------------+--------------+
+| 15         | GND          | 30         | GND          |
++------------+--------------+------------+--------------+
+
+Requirements
+************
+
+Your board needs to have a ``dcmi`` device tree label to work with this shield.
+
+Usage
+*****
+
+The shield can be used in any application by setting ``SHIELD`` to
+``_st_b_cams_omv_mb1683`` and adding the necessary device tree properties.
+
+Set ``--shield "_st_b_cams_omv_mb1683"`` when you invoke ``west build``. For example:
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/subsys/video/capture
+   :board: stm32h7b3i_dk
+   :shield: _st_b_cams_omv_mb1683
```

Comment:
Why using  the prefix `_` in `_st_b_cams_omv_mb1683` ?

Reply:
Fixed. It was a copy past problem! :-)

Thanks for your review! Let me know if you have any other suggestion.

Reply:
For now PR was in draft, so I didn't go much further in the review.If not draft anymore, just move PR to ready for review.

Reply:
I'm wanting to this https://github.com/zephyrproject-rtos/zephyr/pull/76124 going to upstream before change it to review.